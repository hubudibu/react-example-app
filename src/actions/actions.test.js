import { addTodo } from './index.js';

describe('Actions', () => {
  it('match snapshot', () => {
    const action = addTodo('Sample text');
    expect(action).toMatchSnapshot();
  });
});
