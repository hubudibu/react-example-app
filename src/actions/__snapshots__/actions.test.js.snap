exports[`Actions match snapshot 1`] = `
Object {
  "id": 10,
  "text": "Sample text",
  "type": "ADD_TODO",
}
`;
